web/viewer.css

```diff
@@ -1699,6 +1699,15 @@ html[dir="rtl"] #documentPropertiesOverlay .row > * {
 #PDFBug table {
   font-size: 10px;
 }
+#PDFBug table.showText {
+  border-collapse: collapse;
+  text-align: center;
+}
+#PDFBug table.showText,
+#PDFBug table.showText tr,
+#PDFBug table.showText td {
+  border: 1px solid black;
```

Comment:
I'd suggest adding e.g. `padding: 1px` here as well, since that'd improve readability of the table data a little bit (especially for the `originalCharCode` values).